pkgs/by-name/ya/yao-pkg/package.nix

```
@@ -0,0 +1,106 @@
+{
+  autoPatchelfHook,
+  fetchFromGitHub,
+  fetchYarnDeps,
+  fixup-yarn-lock,
+  lib,
+  makeWrapper,
+  nodejs,
+  stdenv,
+  yarn,
+  yarnBuildHook,
+  yarnConfigHook,
+  yarnInstallHook,
+}:
+
+stdenv.mkDerivation rec {
```

Use finalAttrs if possible
```suggestion
stdenv.mkDerivation (finalAttrs: {
```

---

Have applied your recommendations